src/pages/resultsView/fusion/ResultsViewFusionMapper.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ *
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+import * as React from 'react';
+import ResultsViewFusionTable from './ResultsViewFusionTable';
+import { observer } from 'mobx-react';
+import { ResultsViewFusionMapperStore } from './ResultsViewFusionMapperStore';
+import LoadingIndicator from '../../../shared/components/loadingIndicator/LoadingIndicator';
+import { action, computed, makeObservable, observable } from 'mobx';
+import {
+    getOncoKbIconStyle,
+    updateOncoKbIconStyle,
+} from 'shared/lib/AnnotationColumnUtils';
+
+export interface IFusionMapperProps {
+    store: ResultsViewFusionMapperStore;
+}
+
+@observer
+export default class ResultsViewFusionMapper extends React.Component<
+    IFusionMapperProps,
+    {}
+> {
+    @observable mergeFusionTableOncoKbIcons;
+
+    constructor(props: IFusionMapperProps) {
+        super(props);
+        makeObservable(this);
+
+        this.mergeFusionTableOncoKbIcons = getOncoKbIconStyle().mergeIcons;
+    }
+
+    @action.bound
+    handleOncoKbIconToggle(mergeIcons: boolean) {
+        this.mergeFusionTableOncoKbIcons = mergeIcons;
+        updateOncoKbIconStyle({ mergeIcons });
+    }
+
+    @computed get itemsLabelPlural(): string {
+        const count = this.props.store.dataStore
+            .duplicateStructuralVariantCountInMultipleSamples;
+        const structuralVariantsLabel =
+            count === 1 ? 'structural variant' : 'structural variants';
+
+        const multipleStructuralVariantInfo =
+            count > 0
+                ? `: includes ${count} duplicate ${structuralVariantsLabel} in patients with multiple samples`
+                : '';
+
+        return `Structural Variants${multipleStructuralVariantInfo}`;
+    }
+
+    private renderComponents() {
+        const molecularProfileMap = new Map(
```

Comment:
this doesn't seem like it's a map.  It's just an array of entries?   If so, it should just be an array.